src/main/java/frc/robot/OI.java

```diff
@@ -23,7 +24,8 @@
      * <b>Yellow Button</b>: Climber Up<br>
      * <b>Blue Button</b>: Climber Stop<br>
      * <b>D-Pad Down Button</b>: Reverse all<br>
-     * <b>D-Pad Left Button</b>: Reverse IntakeToTransfer and Intake
+     * <b>D-Pad Left Button</b>: Reverse IntakeToTransfer and Intake<br>
+     * <b>Xbox Left Joystick Button</b>: Move to next bar
```

Comment:
That is still inconsistent

Reply:
How should it be then

Reply:
Move To Next Bar

Reply:
Actually, change all of the other to be like this